aiohttp/web_protocol.py

```diff
@@ -330,13 +330,12 @@ def connection_lost(self, exc: Optional[BaseException]) -> None:
             return
         self._manager.connection_lost(self, exc)
 
-        super().connection_lost(exc)
```

Comment:
Basically, the parent class just sets transport to None without closing it. This function never even tried to close it before, so it looks like calling .force_close() is more sensible.

Reply:
Shouldn't it already be closing if we get here?
https://github.com/python/cpython/blob/c4ee4e756a311f03633723445299bde90eb7b79c/Lib/asyncio/selector_events.py#L865

Reply:
I guess it doesn't matter since it will do nothing in that case